cypress/e2e/awx/views/schedules.cy.ts

```diff
@@ -109,34 +110,195 @@ describe('Schedules', () => {
+describe('Schedules - Create', function () {
+  let organization: Organization;
+  let jobTemplate: JobTemplate;
+  let project: Project;
+  let inventory: Inventory;
+  let inventorySource: InventorySource;
+  before(() => {
+    cy.awxLogin();
+  });
+  beforeEach(function () {
+    project = this.globalProject as Project;
+    this.globalInventory;
+    cy.createAwxOrganization().then((o) => {
+      organization = o;
+      cy.createAwxInventory({ organization: organization.id }).then((i) => {
+        inventory = i;
+        cy.createAwxInventorySource(i, project).then((invSrc) => {
+          inventorySource = invSrc;
+        });
+      });
+    });
+    cy.navigateTo('awx', 'schedules');
   });
-  it.skip('can create a simple schedule of resource type Job template', () => {
-    //Make sure to assert the input data in the review step
-    //Survey, prompts, rules and exceptions are not needed for this test
+
+  it('can create a simple schedule and navigate to the schedule details page', () => {
+    cy.createAwxJobTemplate({
+      name: 'E2E Credentials ' + randomString(4),
+      organization: organization.id,
+      project: project.id,
+      inventory: inventory.id,
+    }).then((jt) => {
+      jobTemplate = jt;
+      const scheduleName = 'E2E ' + randomString(4);
+      cy.getBy('[data-cy="create-schedule"]').click();
+
+      cy.selectDropdownOptionByResourceName('node_type', 'Job template');
+      cy.selectDropdownOptionByResourceName('job-template-select', `${jobTemplate.name}`);
+      cy.get('[data-cy="name"]').type(`${scheduleName}`);
+      cy.selectDropdownOptionByResourceName('timezone', 'Zulu');
+      cy.clickButton(/^Next$/);
+      cy.get('[data-cy="interval"]').clear().type('100');
+      cy.selectDropdownOptionByResourceName('freq', 'Hourly');
+      cy.get('[data-cy="count-form-group"]').type('17');
+      cy.get('[data-cy="add-rule-button"]').click();
+      cy.get('tr[data-cy="row-id-1"]').within(() => {
+        cy.get('td[data-cy="rrule-column-cell"]').should(
+          'contains.text',
+          'RRULE:FREQ=HOURLY;INTERVAL=100;WKST=SU'
+        );
+      });
+      cy.clickButton(/^Next$/);
+      cy.clickButton(/^Next$/);
+      cy.get('dl[data-cy="rule-1"]').should('be.visible');
+      cy.clickButton(/^Finish$/);
+      cy.verifyPageTitle(`${scheduleName}`);
+    });
   });
 
-  it.skip('can create a simple schedule of resource type Workflow job template', () => {
-    //Make sure to assert the input data in the review step
-    //Survey, prompts, rules and exceptions are not needed for this test
+  it('can create a simple schedule of resource type Workflow job template', () => {
+    let wfjt: WorkflowJobTemplate;
+    cy.createAwxWorkflowJobTemplate({
+      name: 'E2E Workflow Job Template ' + randomString(4),
+      organization: organization.id,
+      inventory: inventory.id,
+    }).then((jt) => {
+      wfjt = jt;
+      const scheduleName = 'E2E ' + randomString(4);
```

Comment:
we could refining the schedules names per test block. Something like     
`const scheduleName = 'E2E Schedule Create' + randomString(4);`

that would help us to debug faster.